build_scripts/build_protobuf.sh

```diff
@@ -17,7 +17,7 @@
 # protobuf, make two steps for cross compilation if needed
 pushd third_party/protobuf
 ./autogen.sh
-./configure CXXFLAGS="-fPIC" --prefix=/usr/local --disable-shared 2>&1 > /dev/null
+./configure CXXFLAGS="-fPIC" --prefix=${INSTALL_PREFIX} --disable-shared 2>&1 > /dev/null
```

Comment:
```suggestion
./configure CXXFLAGS="-fPIC" --prefix=${HOST_INSTALL_PREFIX} --disable-shared 2>&1 > /dev/null
```

Reply:
As we want to install protoc for host, and bellow we install protobuff for target just saying it should use protoc from host.

Reply:
Makes sense. Fixed in f631458bfd78a835d26c635e3232de1b13176b54.